secp256kfun/src/poly.rs

```diff
@@ -0,0 +1,317 @@
+//! Utilities for working with polynomials on the secp256k1 elliptic curve.
+//!
+//! A polynomial f(x) of degree k is defined by its coefficients
+//!
+//! `f(x) = a_0 + a_1 * x + a_2 * x^2 + ... + a_k * x^k`
```

Comment:
defined by its coefficients `a_0, ... a_n`. The coefficients can be `Scalar` or `Point`.

I wonder if you can use LaTeX in rust doc yet.

Reply:
funnily enough search took me to the zcash frost squad discussing using latex in their rust doc https://github.com/ZcashFoundation/frost/pull/436#discussion_r1265895682, apparently you can using mathjax in html header..